app/assets/javascripts/app.coffee.erb

```diff
@@ -0,0 +1,68 @@
+receta = angular.module('receta',[
+  'templates',
+  'ngRoute',
+  'controllers',
+])
+
+receta.config([ '$routeProvider',
+  ($routeProvider)->
+    $routeProvider
+    .when('/',
+      templateUrl: "<%= asset_path('index.html') %>"
+      controller: 'PcbuilderController'
+    )
+])
+
+components = [
+  {
+    name: 'Processor',
+    category: 'cpu',
+    image: '<%= asset_path("icon-cpu.png") %>'
```

Comment:
If you get the categories out of the db the icon src can be build like "icon-" + category.name ".png"